network-api/networkapi/wagtailpages/templates/wagtailpages/fragments/generic_card.html

```diff
@@ -1,7 +1,7 @@
 {% load wagtailcore_tags wagtailimages_tags %}
 {% if page %}
     {% if not url %}
-        {% pageurl page as url %}
+        {% pageurl page.localized as url %}
```

Comment:
Calling out that we are typically trying to move away from using .localized in templates, however, this template is shared amongst a couple of other pages as well, and the work necessary to get this template to work with all of them without using .localized would exceed the scope of this ticket.

However, we do have issue https://github.com/MozillaFoundation/foundation.mozilla.org/issues/10018 that captures that work!